code/modules/mob/dead/observer/observer.dm

```diff
@@ -413,8 +413,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		if(ghost && !is_admin_level(z))
 			ghost.timeofdeath = world.time
 
-			// Larva queue: We use the larger of their existing queue time or the new timeofdeath except for facehuggers
-			var/new_tod = isfacehugger(src) ? 1 : world.time
+			// Larva queue: We use the larger of their existing queue time or the new timeofdeath except for facehuggers/lesser drones
+			var/new_tod = (isfacehugger(src) || islesserdrone(src)) ? 1 : world.time
```

Comment:
You need to also update this same check in the proc `/mob/proc/ghostize(can_reenter_corpse = TRUE, aghosted = FALSE)`: https://github.com/cmss13-devs/cmss13/pull/3939/files#diff-08897ff01ebb08d739f4a3e87da529ac87080df7a6bfbe17fdfd85d5a2e5b401L370